samples-aspnetcore-3x/self-hosted-login/okta-aspnetcore-mvc-example/Views/Account/Profile.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿@using okta_aspnetcore_mvc_example.Okta
+
+@model ProfileModel
+
+@{
+    var userInfo = Model.UserInfo;
+    var tokens = Model.Tokens;
+}
+
+<h1>Profile</h1>
```

Comment:
If we use Asp.Net identity, we should be able to reuse the same profile view as the existing example: https://github.com/okta/okta-idx-dotnet/blob/idx-auth-layer/src/direct-auth-idx/direct-auth-idx/Views/Home/Profile.cshtml

Reply:
I'll review that branch and implement as you propose.